parachain/runtime/test-common/src/lib.rs

```diff
@@ -288,7 +296,60 @@ pub fn send_transfer_token_message_failure<Runtime, XcmConfig>(
 				destination_address,
 				fee_amount,
 			);
-			// check err is NotHoldingFees
+			assert_err!(outcome.ensure_complete(), expected_error);
+		});
+}
+
+#[allow(clippy::too_many_arguments)]
+pub fn send_transfer_token_message_with_invalid_fee_params<Runtime, XcmConfig>(
```

Comment:
I don't think we need this test - the assumption is that the stored pricing parameters are valid after being set.